src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationUtils.getQualifiedFieldName;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.stereotype.Component;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.trust.TrustDataDto;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.DateValidators;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.StringValidators;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+@Component
+public class TrustDetailsValidator {
```

Comment:
A bit confused, as this validator is not referenced from the `OverseasEntitySubmissionDtoValidator`, so presumably it's not actually wired in yet. Is that intentional?

Reply:
sorry @mwestacott I have pushed the missing missing files: OverseasEntitySubmissionDtoValidator and OverseasEntitySubmissionDtoValidatorTest.